dev-docs/auth.md

```diff
@@ -92,3 +92,26 @@ The resource plucks the `id_token` out of that response.
   (This doesn't mean that we don't pick that route in the future.)
 
 * There's an example Go method in [`docs/examples/api_request.go`](examples/api_request.go).
+
+## Using the Identity and Access Management System in the Automate UI
+By default any new user, team, or non-admin API token is denied access to everything. They all need to be added to policies in order to be granted permissions. 
+
+### Adding a New User to a New Policy
+1. Create an admin token using 
+    ```
+    export TOKEN=`chef-automate iam token create <your-token-name-here> --admin`
```

Comment:
[sanity check] - Not sure if this is just my case, but I wasn't able to run this as a one liner.  

Instead I ran `chef-automate iam token create <your-token-name-here> --admin` and then copied the result and ran `export TOKEN=<result>`

Reply:
ohhh thats so weird, whats the output it gives you when you try and run it as one line?

Reply:
`bash: export: --admin: not a valid identifier`

Reply:
ahh...wait ok..  It works if I use `export TOKEN='chef-automate iam token create sometoken --admin'`

the difference being using apostrophes

Reply:
so the backticks done work? hmm

Reply:
scratch that sorry - apostrophes did not work either

Reply:
i think apostrophes would just save that whole command into token, right?

Reply:
and here I am now and backticks are working.

I think i'm done with Reviews for the day.  Well done!

Reply:
lol okay cool, i wonder what was going on before. glad its working now!

Reply:
I just triple checked using backticks, exactly correct. 🙏 

I think we can put what I was getting back as 100% user error 😂